src/lib/drivers/driver.js

```diff
@@ -92,10 +92,15 @@ class DriverBase {
 
   evaluateAsync(asyncExpression) {
     return new Promise((resolve, reject) => {
+      let asyncTimeout = 0;
```

Comment:
we should set this to undefined and only clear if it is not undefined. The spec does not say that 0 is an invalid ID so potentially this could get us into trouble ... (sorry to be nitty about this)